dtests/src/test/scala/io/snappydata/hydra/SnappyHydraRunner.scala

```diff
@@ -65,7 +65,11 @@ class SnappyHydraRunner extends SnappyTestRunner {
     val c14 = "grep -v java.lang.reflect.InvocationTargetException"
     val c15 = "grep -v org.apache.spark.storage.ShuffleBlockFetcherIterator." +
         "throwFetchFailedException"
-    val command1 = c1 #| c2 #| c3 #| c4 #| c5 #| c6 #| c7 #| c8 #| c12 #| c13 #| c14 #| c15
+    val c16 = "grep -v java.lang.IllegalStateException"
+    val c17 = "grep -v com.gemstone.gemfire.distributed.LockServiceDestroyedException"
+    val c18 = "grep -v com.gemstone.gemfire.GemFireIOException"
+    val command1 = c1 #| c2 #| c3 #| c4 #| c5 #| c6 #| c7 #| c8 #| c12 #| c13 #| c14 #| c15 #|
+        c16 #| c17 #| c18
```

Comment:
I have recently added support for collecting thread dump of all server and lead members in case of failure while starting or stopping them where I check for java.lang.IllegalStateException or waiting or stopping status in stop-all script output. And as a result of grep command for exceptions in the smoke bt logs, it shows this exception found  in logs as it was not added in the expected list and mark the bt as failed.

as I said, there are lot many different kinds of exceptions thrown in case of HA and the list is not definite one, So I had no way to know the list of expected exceptions to mark the test as pass or fail.
Sonal is already working on catching all the exceptions got while executing queries through spark app and snappy job and then report them back to hydra once the test run is done in order to mark the test as pass or fail. But we are yet to receive expected list of exceptions in case of HA.

We are going to work on this task on high priority after 1.0